Makefile

```diff
@@ -302,6 +302,9 @@ test_log_file := test_$(python_version_fn).log
 # - 50571 dparse (user safety) 0.4.1 -> 0.5.2, 0.5.1 -> 0.5.2.  ReDos issue
 # - 50885 Pygments 2.7.4 cannot be used on Python 2.7
 # - 50886 Pygments 2.7.4 cannot be used on Python 2.7
+# - 51159 cryptography  v 39.0.0 drops support for C library "LibreSSL" < 3.4
+# - 50885 pygments Lexer infinite loop in versions Pygments versions 1.5 to 2.7.3
+# - 50886 pygments regular expressions vulnerable to ReDos, versions >=1.1,<2.7.4
```

Comment:
We should only put those safety issues on this list that we add to the ignore list of the safety command.

Plus, the comment should say why we cannot upgrade to the version that solves the safety issue, and not repeat the safety issue.

Plus, we should always try to upgrade to the version that fixes the issue, unless we cannot do that for some Python version. Which means the minimum-constraints.txt file and the requirements file should be updated to specify the package version that fixes the safety issue. If needed, this needs to be done dependent on the Python version.